models/Translation.php

```diff
@@ -513,6 +513,11 @@ public static function importTranslationsFromFile(string $file, $domain = self::
                             $t->save();
                         }
                     }
+
+                    // call the garbage collector if memory consumption is > 100MB
+                    if (memory_get_usage() > 100000000) {
```

Comment:
What about writing it like this for better readability?

```suggestion
                    if (memory_get_usage() > 100_000_000) {
```